docs/installing-the-extension.md

```diff
@@ -106,11 +106,17 @@ not from the `main` branch: run `git checkout v5.2` in both `Ed-Fi-ODS` and
     Initdev
     ```
 
+When you run the Web API, look at the base endpoint, which has metadata about
+your installation. Confirm that LMSX is listed in the data models. If not seen,
+please review the steps above. Particularly check that you selected the correct
+"PackageVersion" in step 2, matching the version of code that you are building
+(5.2.x or 5.3.x).
+
 ### Dynamic Plugin Into Runtime
```

Comment:
This section requires a step similar to:
1. Copy [lmsx.ps1](../extension/lmsx.ps1) to your
   `Ed-Fi-ODS-Implementation/Plugin` directory.

As in the *Dynamic Plugin Into Source Code* section. This script should be copied to the Plugin folder of the published IIS Website before running.